packages/contracts-bedrock/deploy-config/baklava.json

```diff
@@ -0,0 +1,77 @@
+{
+  "l1StartingBlockTag": "0x382262bec314539f272adb4cdc8165f14019586a87a4301402436c7894eefa47",
+  "l1ChainID": 17000,
+  "l2ChainID": 62320,
+  "l2BlockTime": 1,
+  "l1BlockTime": 12,
+  "maxSequencerDrift": 600,
+  "sequencerWindowSize": 7200,
+  "channelTimeout": 300,
```

Comment:
With the results from https://github.com/celo-org/celo-blockchain-planning/issues/804, I think we can leave this at the default (50). Note: so far nobody review my reasoning in that issue.